### PR TITLE
Add support for webhook session ID

### PR DESCRIPTION
**Pull request by wangdaliu**

### CODE REVIEW

File: affirm/src/main/java/com/affirm/android/model/Metadata.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    @SerializedName("mode")
+    public abstract String mode();
+
+    @AutoValue.Builder
+    public abstract static class Builder {
+        public abstract Builder setShippingType(String value);
+
+        public abstract Builder setEntityName(String value);
+
+        public abstract Builder setPlatformType(String value);
```
Comment (kevindavmoore) [inline affirm/src/main/java/com/affirm/android/model/Metadata.java]: Will platform type be anything other than Affirm Android SDK? If not, don't let the user change. Also consider having it as a default value

Comment (diptib10) [reply affirm/src/main/java/com/affirm/android/model/Metadata.java]: It shouldn't change, so agree to make it as a default value. Same applies to platform_affirm.

Comment (wangdaliu) [reply affirm/src/main/java/com/affirm/android/model/Metadata.java]: Fixed

File: affirm/src/main/java/com/affirm/android/CheckoutRequest.java [hunk 2]
```diff
@@ -76,16 +76,17 @@ public void create() {
...
-        metadataJson.addProperty("platform_affirm", BuildConfig.VERSION_NAME);
 
         final JsonObject checkoutJson = jsonParser.parse(gson.toJson(checkout)).getAsJsonObject();
 
         checkoutJson.add("merchant", merchantJson);
         checkoutJson.addProperty("api_version", "v2");
+
+        final JsonObject metadataJson =
+                jsonParser.parse(gson.toJson(checkout.metadata())).getAsJsonObject();
+        metadataJson.addProperty("platform_type", "Affirm Android SDK");
```
Comment (kevindavmoore) [inline affirm/src/main/java/com/affirm/android/CheckoutRequest.java]: Have all strings be constants

**Comment by AmyBeall** [comment]
@wangdaliu please resolve these conflicts

**Comment by wangdaliu** [comment]
@AmyBeall Conflict resolved

**Comment by AmyBeall** [comment]
@wangdaliu It appears more conflicts were created when I merged the other branch

**Comment by wangdaliu** [comment]
@AmyBeall Conflict resolved

**Pull request merged**